cmd/cassandra-writer/main.go

```diff
@@ -156,6 +162,21 @@ func newService(session *gocql.Session, logger logger.Logger) consumers.Consumer
 	return repo
 }
 
+func makeTransformer(cfg config, logger logger.Logger) transformers.Transformer {
```

Comment:
whould it make sense that this function is actually in Transformer interface?

Reply:
This can't be extracted to the Transformer interface because it would cause circular dependency (implementation imports Transformer to implement it and Transformer imports implementations so that it can instantiate them).